packages/core/src/routes/interaction/utils/sign-in-experience-validation.ts

```diff
@@ -59,7 +59,7 @@ export const identifierValidation = (
         if (
           'passcode' in identifier &&
           !verificationCode &&
-          ![SignUpIdentifier.Email, SignUpIdentifier.EmailOrSms].includes(signUp.identifier) &&
```

Comment:
I don't think these expressions are equal. Should probably be:
```ts
!signUp.identifiers.includes(SignInIdentifier.Email) && !signUp.identifiers.includes(SignInIdentifier.EmailOrSms) &&
```

Reply:
This is new logic with a new type.
We don't have a `SignInIdentifer.EmailOrSms`, we use `includes(SignInIdentifier.Email)` to know the Sign-Up Email verification is enabled.